src/Lykke.Service.Decred.Api/Controllers/TransactionController.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.Net;
+using System.Threading.Tasks;
+using Common.Log;
+using Lykke.Service.BlockchainApi.Contract.Common;
+using Lykke.Service.BlockchainApi.Contract.Transactions;
+using Lykke.Service.Decred.Api.Common;
+using Lykke.Service.Decred.Api.Services;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Lykke.Service.Decred.Api.Controllers
+{
+    public class TransactionController : Controller
+    {
+        private readonly ILog _log;
+        private readonly IAddressValidationService _addressValidationService;
+        private readonly IUnsignedTransactionService _txBuilderService;
+        private readonly ITransactionBroadcastService _txBroadcastService;
+
+        public TransactionController(
+            ILog log,
+            IAddressValidationService addressValidationService,
+            IUnsignedTransactionService txBuilderService,
+            ITransactionBroadcastService txBroadcastService)
+        {
+            _log = log;
+            _addressValidationService = addressValidationService;
+            _txBuilderService = txBuilderService;
+            _txBroadcastService = txBroadcastService;
+        }
+
+        [HttpGet("api/capabilities")]
+        public CapabilitiesResponse GetCapabilities()
+        {
+            return new CapabilitiesResponse
+            {
+                AreManyInputsSupported = false,
+                AreManyOutputsSupported = false,
+                IsTransactionsRebuildingSupported = false
+            };
+        }
+
+        [HttpPost("api/transactions/single")]
+        public async Task<IActionResult> BuildSingleTransaction([FromBody] BuildSingleTransactionRequest request)
+        {
+            // Do not scale the fee
+            const int feeFactor = 1;
+            return await BuildTxInternal(request, feeFactor);
+        }
+
+        private async Task<IActionResult> BuildTxInternal(BuildSingleTransactionRequest request, decimal feeFactor)
```

Comment:
This will not allow me to return "notEnoughBalance" and "amountIsTooSmall" errors.  This directly conflicts with the requirements I've been following?